python/ray/serve/api.py

```diff
@@ -890,6 +894,57 @@ async def __del__(self):
     return decorator
 
 
+@PublicAPI(stability="alpha")
+class DAGHandle:
+    """Resolved from a DeploymentMethodNode at runtime.
+
+    This can be used to call the DAG from a driver deployment to efficiently
+    orchestrate a multi-deployment pipeline.
+    """
+
+    def __init__(self, serialized_dag_json: str):
+        raise NotImplementedError()
+
+    def remote(self, *args, **kwargs) -> ray.ObjectRef:
+        """Call the DAG."""
+        raise NotImplementedError()
+
+
+@PublicAPI(stability="alpha")
+class DeploymentMethodNode(ClassNode):
```

Comment:
this subclasses DAGNode. I think we can just import the existing one ?

Reply:
IMO the existing one should be moved here w/ other top-level Serve APIs.

Reply:
im good with that and it helps to resolve circular imports

Reply:
yup. let's plan to do that.